### DISCUSSION TITLE
Preset Sunset

### DISCUSSION THREAD

**Discussion by alexilk** [Akifyss/obsidian-border#135] [Q&A]
Hi, thanks for this wonderful theme. I'm just wondering why the middle pane is not blurred under the preset Sunset like the preview. I've reset it before importing the code, and the backdrop filter has been set to be `blur(32px)` by default.

![image](https://github.com/Akifyss/obsidian-border/assets/154331555/0db6d756-b27d-4402-97c3-077357b208c6)

Many thanks in advance.

**Comment by Akifyss** [comment]
Hello, this issue may be related to the display scaling of the Windows. Apart from changing the display scaling, there is currently no other solution available. Sorry.
![image](https://github.com/Akifyss/obsidian-border/assets/23567154/12a53042-9f9b-4334-8b59-cc9413719a32)

**Comment by robbiecarlton** [comment]
On mac, just doing a "cmd" + "+" and then "cmd" + "-" to temporarily adjust the scaling fixed this for me
